tooling/scripts/test-categories.yaml

```diff
@@ -84,6 +84,7 @@ group-04:
   - mongodb-grouped
   - mybatis
   - sjms-artemis-client
+  - observability-services
```

Comment:
I've put this where there was the `micrometer`. However not sure which is the correct group to choose from :/